modules/op-building-images-for-tekton-hub.adoc

```diff
@@ -0,0 +1,64 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="building-images-for-tekton-hub_{context}"]
+= Building images for {tekton-hub}
+
+[role="_abstract"]
+Before deploying {tekton-hub}, you can build and push images to a registry.
+
+You can use images from the Quay registry and skip the steps for building the images. If you use images from Quay, use the images corresponding to the {tekton-hub} version you are deploying. 
+
+For example, if you are deploying {tekton-hub} 1.7.2, use the images that has the `v1.7.2` tag:
+
+* UI image: link:https://quay.io/tekton-hub/ui:v1.7.2[]
+* API image: link:https://quay.io/tekton-hub/api:v1.7.2[]
+* Database migration image: link:https://quay.io/tekton-hub/db-migration:v1.7.2[]
+
+You can find the list of {tekton-hub} releases on the link:https://github.com/tektoncd/hub/releases[release page].
+
+[discrete]
+.Prerequisites
+
+* Ensure you are in the project root and signed in to your image registry account.
+* Docker or Podman is installed.
+
+[discrete]
+.Procedure
+
+. Build and push the {tekton-hub} UI image.
++
+[source,terminal]
+----
+$ docker build -f images/ui.Dockerfile \ <1>
+  -t <image> . \ <2>
+  && docker push <image>
+----
+<1> Path to the dockerfile for the UI image.
+<2> Substitute with the registry and image name. For example, if you are using Quay as the image registry, use the `quay.io/<username>/ui` format.     
```

Comment:
```suggestion
<2> Substitute with the registry and image name. For example, `quay.io/<username>/ui`.     
```
The current text is good and clear, but you could simplify it like this. (multiple instances)